miden/src/cli/verify.rs

```diff
@@ -0,0 +1,61 @@
+use super::data::{InputFile, OutputFile, ProgramHash, ProofFile};
+use log::debug;
+use std::path::PathBuf;
+use std::time::Instant;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "Verify", about = "Verify a miden program")]
+pub struct VerifyCmd {
+    /// Path to .masm script file
+    #[structopt(short = "s", long = "script-file", parse(from_os_str))]
+    script_file: PathBuf,
```

Comment:
so in this case `script_file` is used to determine default files names - e.g. `<script_file>.inputs`, `<script_file>.outputs`, `<script_file>.proof`.  It is not used to interact with the script / produce program hash.  We could remove `script_file` entirely and make `input_file`, `output_file` and `proof_file` required arguments as opposed to optional.  What do you think?

Reply:
Ah - I see! This could work - but wonder if we could just make `proof_file` required and then infer input/output files based on that. What do you think?